lib/db/DB.ts

```diff
@@ -58,8 +58,12 @@ class DB {
       const { dialectName } = this.sequelize.connectionManager;
       this.logger.info(`connected to database. host:${host} port:${port} database:${database} dialect:${dialectName}`);
     } catch (err) {
-      this.logger.error('unable to connect to the database', err);
-      throw err;
+      if (err instanceof Sequelize.ConnectionError) {
```

Comment:
This returns true for various of other errors, besides when database does not exist.
You should inspect it in more details for the right err code/msg.

Reply:
I am not sure how to achieve this but I've tried skimming through sequelize docs to check for any error code to capture but no luck. can you guide me on this?

Reply:
Sure.
I won't leave this PR open because of this, so let's talk about this today and finish it by tomorrow.